authappliance/menu.py

```diff
@@ -306,8 +314,8 @@ def stop_redundancy(self):
         self.display_messages()
 
     def _execute_local_sql(self, sql):
-        p = Popen(['mysql', '--defaults-extra-file=/etc/mysql/debian.cnf'],
-                  stdin=PIPE, stdout=PIPE, stderr=PIPE)
+        p = my_Popen(['mysql', '--defaults-extra-file=/etc/mysql/debian.cnf'],
+                     stdin=PIPE, stdout=PIPE, stderr=PIPE, encoding='utf8')
```

Comment:
Since we use the `encoding` parameter with the same value everywhere, we could add `'utf8'` as the default value of the parameter.

Reply:
Makes sense. Will do.

Reply:
done